lib/rules/lint-attribute-indentation.js

```diff
@@ -275,8 +280,8 @@ module.exports = class AttributeSpacing extends Rule {
       expectedLineStart = node.loc.start.line + 1;
     }
 
-    this.iterateParams(node.params, 'positional', expectedLineStart, expectedColumnStart, node);
-    this.iterateParams(node.hash.pairs, 'attribute', expectedLineStart + node.params.length, expectedColumnStart, node);
+    expectedLineStart = this.iterateParams(node.params, 'positional', expectedLineStart, expectedColumnStart, node);
```

Comment:
My supervisor did mention that it is probably a bad choice to make ```iterateParams``` return ```expectedLineStart```, since most other function have no return values and it needlessly entangles them.
Do you see another way to make this work?